content/doc/book/system-administration/security/matrix-based-security.adoc

```diff
@@ -0,0 +1,133 @@
+---
+title: Matrix Based Security
```

Comment:
This page needs an entry added to the `_chapter.yml` file so that it will appear in the contents list on the left.  Without that entry, it is difficult for the reader to find the page, since it is only accessible through the hyperlink in the security.adoc file.

Reply:
This page is in a subfolder `security`, together with `build_authorization.adoc` and `configuring-content-security-policy.adoc`. Adding it to `_ chapter.yml` place them all at the same level. See screenshot below. Is this ok, or can we create another level in the table of contents? 

<img width="282" alt="Screenshot 2020-06-11 at 21 18 49" src="https://user-images.githubusercontent.com/6340362/84429933-43d6ab00-ac29-11ea-8b43-bae61db2034a.png">

Reply:
I think that they should be at the same level in the table of contents.  I'd prefer it be placed immediately below "Securing Jenkins" so that the two related pages are adjacent to one another in the table of contents, but am also OK if they are not adjacent.

Reply:
No problem. I'll make the changes.